xml-app/build.gradle.kts

```diff
@@ -0,0 +1,141 @@
+/*
+ *  Copyright (C) 2023 Rajesh Hadiya
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+    id("dagger.hilt.android.plugin")
+    id("androidx.navigation.safeargs.kotlin")
+    kotlin("kapt")
+}
+
+android {
+    namespace = "com.example.xml_app"
```

Comment:
Can you set the package name to com.hadiyarajesh.xml_app? (For whole app, not just in build file)